packages/next/lib/is-serializable-props.ts

```diff
@@ -64,7 +64,7 @@ export function isSerializableProps(
         page,
         method,
         path,
-        '`undefined` cannot be serialized as JSON. Please use `null` or omit this value all together.'
+        '`undefined` cannot be serialized as JSON. Please use `null` or omit this value altogether.'
```

Comment:
I think this error can be simplified:

```suggestion
        '`undefined` cannot be serialized as JSON. Please use `null` or omit this value.'
```

Reply:
I applied these changes but also had to update the snapshots using the `-u` flag. This also reformatted some of the code, is that alright?

Reply:
My bad, I messed something up while reverting local changes. This should be alright now. I can squash commits once the changes are okay.